tools/validate/src/builds/index.ts

```diff
@@ -3,17 +3,16 @@
  */
 
 /* eslint-disable no-console */
-import { getAllPackages, getPackageName } from '@lg-tools/meta';
+import { getAllPackages, getPackageJson } from '@lg-tools/meta';
 import chalk from 'chalk';
 import fse from 'fs-extra';
 import path from 'path';
-import vm from 'vm';
 
 import { ValidateCommandOptions } from '../validate.types';
 
-import { ModuleType } from './modules.types';
+import { getModuleTypes } from './getModuleTypes';
 
-// A list of
+// A list of packages to ignore
```

Comment:
Can you add a note on why we ignore some packages?

Reply:
that package doesn't actually exist. rm'ing the ignore array